packages/pluggableWidgets/sidebar-web/src/utils/SidebarToggleRegistration.ts

```diff
@@ -0,0 +1,13 @@
+export function registerSidebarToggle(toggle: () => void): () => void {
+    const sidebarToggleLocation = "com.mendix.widgets.web.sidebar.toggle";
+
+    if ((window as any)[sidebarToggleLocation]) {
+        throw Error(
+            "Multiple sidebar widgets on a page isn't supported. Please model your page in such a way that there is just one sidebar."
+        );
```

Comment:
Reminder for future: we do want to support this but the behaviour will be that we toggle all sidebars, check JIRA stories for more info

Reply:
I saw it at the end of yesterday. I think it's a good thing to discuss, because I asked Chris yesterday about it and he looked for a consistency check to only allow one sidebar on a page. I noted to him that we want to prevent pluggable widget specific logic inside Studio Pro as much as possible. I opted to go for a midway solution right now by doing a check on runtime.

Reply:
Will do